include/phasar/PhasarLLVM/TypeHierarchy/LLVMVFTable.h

```diff
@@ -84,6 +85,9 @@ class LLVMVFTable : public VFTable<const llvm::Function *> {
   end() const {
     return VFT.end();
   };
+
+  [[nodiscard]] static std::vector<const llvm::Function *>
+  getVFVectorFromIRVTable(const llvm::ConstantStruct *);
```

Comment:
```suggestion
  [[nodiscard]] static std::vector<const llvm::Function *>
  getVFVectorFromIRVTable(const llvm::ConstantStruct &);
```
The impl does not work with a nullptr and also does not check for it, so making this clear through the API by passing a reference would be a good idea.

Reply:
Yeah makes sense, I'll make that change not in github though.